src/core/meta.ml

```diff
@@ -74,7 +83,26 @@ let get_all () =
 		if d <> Last then d :: loop (i + 1)
 		else []
 	in
-	loop 0
+
+	(* TODO: find a better way to merge with above loop *)
```

Comment:
Someone has a suggestion for this?
- [ ] Rework that part or remove comment

Reply:
You could work with an accumulator again, initializing it to the result of `Hashtbl.fold`. That also has the advantage of being tail-recursive, unlike `d :: loop` which isn't.

Note though that this affects order, not sure if that matters here.

Reply:
Like that? https://github.com/HaxeFoundation/haxe/pull/10858/commits/4483e6db504e5e085e9dc5a86260c6d7a1343b7c